src/main/java/by/onliner/dto/ShoppingCartModalPopup.java

```diff
@@ -0,0 +1,33 @@
+package by.onliner.dto;
+
+import by.onliner.page.AbstractPage;
+import by.onliner.page.MainPage;
+import by.onliner.page.ShoppingCartPage;
+import by.onliner.util.Waiter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ShoppingCartModalPopup extends AbstractPage {
```

Comment:
Why a Popup extends AbstractPage?  In my opinion, popup and page are different things...

Reply:
Created new AbstractComponent class and my components now extending this class

Reply:
Add AbstractComponent and extend it to Menu and ShoppingCartModalPopup